identity-service/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "@audius/libs": "1.2.1",
-    "@solana/web3.js": "^1.2.5",
+    "@optimizely/optimizely-sdk": "^4.6.0",
```

Comment:
@hareeshnagaraj  / @cheran-senthil we have 2 solana/web3.js's in this package.json. It was cleared out by npm install. is this ok?

Reply:
as long as there is 1 we should be fine - thanks for catching